src/mlpack/methods/logistic_regression/logistic_regression_function.hpp

```diff
@@ -28,14 +29,33 @@ template<typename MatType = arma::mat>
 class LogisticRegressionFunction
 {
  public:
+  /**
+   * Construct the LogisticRegressionFunction class with the given labeled
+   * training data.  Optionally, specify lambda, which is the penalty parameter
+   * for L2-regularization.  If not specified, it is set to 1,
+   *
+   * @param predictors Input training variables.
+   * @param responses Outputs results from input training variables.
+   * @param lambda L2-regularization parameter.
+   */
   LogisticRegressionFunction(const MatType& predictors,
                              const arma::Row<size_t>& responses,
-                             const double lambda = 0);
+                             const double lambda = 1);
```

Comment:
If we change the default here we should probably document it in `HISTORY.txt`. Any particular reason for the change?